packages/cli/index.js

```diff
@@ -337,6 +337,13 @@ export async function useSpace(did) {
  */
 export async function spaceInfo(opts) {
   const client = await getClient()
+  if (typeof opts.space === 'boolean') {
```

Comment:
when does this get called? 

also, the error message here is wrong - `--space` does not require a value, it defaults to the currently selected space unless you specify a DID. I'd remove this whole clause - if space is given as a boolean it will be caught below in the other new check you added